app/assets/stylesheets/partials/_posts.scss

```diff
@@ -53,42 +53,41 @@
   .description {
     font-style: italic;
   }
-}
 
-form {
-  @include span-columns(4);
-  @include shift(4);
-
-  text-align: center;
-  text-transform: capitalize;
+  form {
+    @include span-columns(4);
+    @include shift(4);
 
-  input {
-    display: inherit;
-    margin: inherit auto;
     text-align: center;
+    text-transform: capitalize;
+
+    input {
+      margin: inherit auto;
+      text-align: center;
+    }
   }
-}
 
-.search-form {
-  @include span-columns(6);
-  margin-left: 0;
-  width: 100%;
-}
+  .search-form {
```

Comment:
any idea what the dark green highlighting is? I wasn't able to find a quick answer for this from github docs